<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDPSender</class>
 <widget class="QDialog" name="UDPSender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDPSender</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblPath">
       <property name="text">
        <string>File Path</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lePath"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblSrcIPAddress">
       <property name="text">
        <string>Src IPv4 Addr</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leSrcIPAddress"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblDstIPAddress">
       <property name="text">
        <string>Dst IPv4 Addr</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leDstIPAddress"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblSrcPort">
       <property name="text">
        <string>Src Port</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leSrcPort"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblDstPort">
       <property name="text">
        <string>Dst Port</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="leDstPort"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>425</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnSend">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
